README.md

```diff
@@ -79,83 +80,248 @@ Now it's your turn to write SQL queries to achieve the following results (You ne
-    ```
-        Your query
+5.  In this part you need to create a transaction with the following steps:
+
+        a. First, get the ammount for the account `3b79e403-c788-495a-a8ca-86ad7643afaf` and `fd244313-36e5-4a17-a27c-f8265bc46590` after all their movements.
+        b. Add a new movement with the information:
+            from: `3b79e403-c788-495a-a8ca-86ad7643afaf` make a transfer to `fd244313-36e5-4a17-a27c-f8265bc46590`
+            mount: 50.75
+
+        c. Add a new movement with the information:
+            from: `3b79e403-c788-495a-a8ca-86ad7643afaf`
+            type: OUT
+            mount: 731823.56
+
+            * Note: if the account does not have enough money you need to reject this insert and make a rollback for the entire transaction
+
+            ```
+            DECLARE
+        saldo_origen NUMERIC;
+        saldo_despues_transfer NUMERIC;
+
+    BEGIN
+    BEGIN;
+    -- Calcular el saldo actual de la cuenta origen
+    WITH user_balances AS (
+    SELECT
+    a.user_id,
+    SUM(CASE
+    WHEN m.type = 'IN' THEN m.mount
+    WHEN m.type = 'TRANSFER' AND m.account_to = a.id THEN m.mount
+    WHEN m.type = 'TRANSFER' AND m.account_from = a.id THEN -m.mount
+    WHEN m.type = 'OUT' THEN -m.mount
+    WHEN m.type = 'OTHER' AND m.account_from = a.id THEN -m.mount
+    WHEN m.type = 'OTHER' AND m.account_to = a.id THEN m.mount
+    ELSE 0
+    END) + a.mount AS total_balance
+    FROM
+    accounts a
+    LEFT JOIN
+    movements m ON (m.account_from = a.id OR m.account_to = a.id)
+    GROUP BY
+    a.user_id, a.mount
+    )
+    SELECT total_balance INTO saldo_origen
+    FROM user_balances
+    WHERE user_id = '3b79e403-c788-495a-a8ca-86ad7643afaf';
+
+        IF saldo_origen < 50.75 THEN
+            RAISE EXCEPTION 'Saldo insuficiente para la transferencia de 50.75';
+        END IF;
+
+        INSERT INTO movements (id, account_from, account_to, type, mount, created_at)
+        VALUES (gen_random_uuid(), '3b79e403-c788-495a-a8ca-86ad7643afaf', 'fd244313-36e5-4a17-a27c-f8265bc46590', 'TRANSFER', 50.75, CURRENT_TIMESTAMP);
+
+        saldo_despues_transfer := saldo_origen - 50.75;
+
+        IF saldo_despues_transfer < 731823.56 THEN
+            RAISE EXCEPTION 'Saldo insuficiente para el movimiento OUT de 731823.56';
+        END IF;
+
+        INSERT INTO movements (id, account_from, account_to, type, mount, created_at)
+        VALUES (gen_random_uuid(), '3b79e403-c788-495a-a8ca-86ad7643afaf', NULL, 'OUT', 731823.56, CURRENT_TIMESTAMP);
+
+        COMMIT;
+
+    EXCEPTION
+    WHEN OTHERS THEN
+    ROLLBACK;
+    RAISE;
+    END $$;
     ```
 
-    e. How much money the account `fd244313-36e5-4a17-a27c-f8265bc46590` have:
+        d. Put your answer here if the transaction fails(YES/NO):
+        ```
+           YES
+        ```
+
+        e. If the transaction fails, make the correction on step _c_ to avoid the failure:
+        ```
+        -- Cambiar el monto a uno que no exceda el saldo
```

Comment:
don't mix Spanish comments with English comments, try keeping the project's conventions.